### PR TITLE
feat: Add ability for a user to see the repos they have voted for

### LINKED ISSUE #124: Feature: Add ability for a user to see the repos they have voted for
### Type of feature

🍕 Feature

### Current behavior

Currently there is no way for a user to see the repos that they have voted for.

### Suggested solution

Add a `My Votes` link to the secondary navbar that filters the repos based on the repos they have voted for. 

Screen Shot 2022-02-12 at 11 47 47 AM

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by jasonericdavis**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds the feature #124 Add ability for a user to see the repos they have voted for. 

## Related Tickets & Documents

closes #124

## Mobile & Desktop Screenshots/Recordings

Modified the Secondary Navbar to add the new `MyVotes` link
Screen Shot 2022-02-12 at 11 47 47 AM

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
It's my son's birthday
Happy Birthday

### CODE REVIEW

**Comment by 0-vortex** [comment]
Oh boy this looks good! :D

**Comment by 0-vortex** [comment]
Also, happy birthday to your son! 🍕

File: src/lib/database.js [hunk 1]
```diff
@@ -72,3 +72,27 @@ export async function fetchRecommendations(orderBy = 'total_stars') {
...
+   * do joins when a foreign key exists
+  */
+  const { data: votedRepose, error } = await supabase
+    .from('recommendations')
+    .select()
+    .in('repo_name', votes.map((v) => v.repo_name))
+    .order('votes', { ascending: false });
+
+  if (error) console.error(error);
+  return votedRepose;
```
Comment (mtfoley) [inline src/lib/database.js:97]: For variable name on this and other lines, you may have meant “votedRepos”.

Comment (jasonericdavis) [reply src/lib/database.js]: Good catch. I updated the variable name.

**Review by bdougie — COMMENTED** [review]
LGTM. Thanks for taking this on.

**Pull request merged**